assets/airports/eidw.json

```diff
@@ -875,136 +1072,578 @@
+            "draw": [
+                ["DW554", "DW555", "DEXEN"],
+                ["OE", "_DEXEN1", "_DEXEN2", "_DEXEN3", "_DEXEN4", "_DEXEN5", "_DEXEN6", "_DEXEN7", "_DEXEN8", "_DEXEN9", "_DEXEN10", "_DEXEN11", "_DEXEN12", "_DEXEN13", "_DEXEN14", "_DEXEN15", "_DEXEN16", "_DEXEN17", "_DEXEN18", "_DEXEN19", "_DEXEN20", "_DEXEN21", "_DEXEN22", "_DEXEN23", "_DEXEN24", "_DEXEN25", "_DEXEN26", "_DEXEN27", "_DEXEN28", "_DEXEN29", "_DEXEN292", "DEXEN"],
+                ["DWS81", "DEXEN"]
+            ]
         },
         "INKUR": {
             "icao": "INKUR",
             "name": "Inkur",
-            "suffix": {"10":"6E", "28":"5A", "16":"5M", "34":"1G"},
+            "suffix": {"10":"6E", "16":"5M", "28":"5A", "34":"1G"},
             "rwy": {
-                "10": [["DW551", "A30+"], "LEMTA", "DODIG"],
-                "28": [["OE", "A30++"]],
-                "16": [["DWS77", "A30+"], "DWS78", "NITIL"],
-                "34": [["DWN26", "A30+"], "DWW06"]
+                "10": [["^DW551", "A30+"], "LEMTA", "DODIG"],
+                "16": [["^DWS77", "A30+"], "DWS78", "NITIL"],
+                "28": [["^OE", "A30+"]],
+                "34": [["^DWN26", "A30+"], "DWW06"]
+            },
+            "body": [],
+            "exitPoints": {
+                "INKUR": ["INKUR"]
             },
-            "body": ["INKUR"],
-            "draw": [["LEMTA", "DODIG", "INKUR"], ["OE", "INKUR"], ["DWS77", "DWS78", "NITIL", "INKUR"], ["DWN26", "DWW06", "INKUR"]]
+            "draw": [
+                ["LEMTA", "DODIG", "INKUR"],
+                ["_LIFFY5", "INKUR"],
+                ["DWS77", "DWS78", "NITIL"],
+                ["DWN28", "INKUR"],
+                ["_BAL11", "DWW06", "INKUR"]
+            ]
         },
-        "DEXEN": {
-            "icao": "DEXEN",
-            "name": "Dexen",
-            "suffix": {"10":"2E", "28":"1A", "16":"1M"},
+        "INKUR(AB)": {
+            "icao": "INKUR(AB)",
+            "name": "Inkur",
+            "suffix": {"10":"6F", "16":"5P", "28":"5B"},
             "rwy": {
-                "10": [["DW554", "A30+"], "DW555"],
-                "28": [["OE", "A30+"]],
-                "16": [["DWS77", "A30+"], "DWS81"]
+                "10": ["DW931", "LEMTA", "DODIG", "INKUR"],
+                "16": ["OE"],
+                "28": ["DUB", "DWN28"]
+            },
+            "body": [],
+            "exitPoints": {
+                "INKUR": ["INKUR"]
             },
-            "body": [["DEXEN", "A120+"]],
-            "draw": [["DW554", "DW555", "DEXEN"], ["OE", "DEXEN"], ["DWS81", "DEXEN"]]
+            "draw": [
+                ["INKUR"]
+            ]
+        },
+        "KISHA": {
+            "icao": "KISHA",
+            "name": "Keeshah",
+            "suffix": {"10":"2F", "16":"1P", "28":"1B", "34":"1H"},
+            "rwy": {
+                "10": ["DW931"],
+                "16": ["OP", "BATED"],
+                "28": ["DUB"],
```

Comment:
Hm. Kinda disconcerting that our various data sources aren't in agreement, but in the end, what can we do... we're pretty close to being right so it'll have to do.

And you appear to be right about the waypoints. Not sure why the jepps are showing what they are, but I pulled the actual navdata installed from navigraph for one of my aircraft, and it shows that which is congruent with navigraph native charts and the IAIP. 

```
KISH1B,28,,,,0,1,279,1,0,0,0,0,0,1,750,000,0.0,000,000,0,
KISH1B,28,DUB,+53.499389,-006.307111,0,1,R000,0,0,0,0,0,0,0,0,000,0.0,000,000,0,
KISH1B,28,KISHA,+53.483806,-005.639333,0,0,000,0,0,0,0,0,0,0,0,000,0.0,000,000,0,
KISH1B,28,LIFFY,+53.480083,-005.500000,0,0,000,0,0,0,0,0,0,0,0,000,0.0,000,000,0,
```

So that sounds like the one we ought to go with. You can leave it as you had it. :shipit: